packages/fxa-payments-server/src/components/CouponForm/index.stories.tsx

```diff
@@ -3,19 +3,77 @@ import React, { useState } from 'react';
 
 import CouponForm from '.';
 import MockApp from '../../../.storybook/components/MockApp';
-import * as Coupon from 'fxa-shared/dto/auth/payments/coupon';
+import {
+  CouponDetails,
+  couponDetailsSchema,
```

Comment:
Nit: This isn't being used anymore, so can you remove it?

We have a VS Code extension called TS Hero that can do these things for you automatically e.g. on command or on save. It will also sort the imports alphabetically. From the Command Palette > TS Hero: Organize imports (sort and remove unused).

Reply:
done